pythonforandroid/python.py

```diff
@@ -113,70 +113,42 @@ def get_recipe_env(self, arch=None, with_flags_in_cc=True):
                 arch=arch, with_flags_in_cc=with_flags_in_cc)
 
         env = environ.copy()
-
-        android_host = env['HOSTARCH'] = arch.command_prefix
-        toolchain = '{toolchain_prefix}-{toolchain_version}'.format(
-            toolchain_prefix=self.ctx.toolchain_prefix,
-            toolchain_version=self.ctx.toolchain_version)
-        toolchain = join(self.ctx.ndk_dir, 'toolchains',
-                         toolchain, 'prebuilt', build_platform)
-
-        env['CC'] = (
-            '{clang} -target {target} -gcc-toolchain {toolchain}').format(
-                clang=join(self.ctx.ndk_dir, 'toolchains', 'llvm', 'prebuilt',
-                           build_platform, 'bin', 'clang'),
-                target=arch.target,
-                toolchain=toolchain)
-        env['AR'] = join(toolchain, 'bin', android_host) + '-ar'
-        env['LD'] = join(toolchain, 'bin', android_host) + '-ld'
-        env['RANLIB'] = join(toolchain, 'bin', android_host) + '-ranlib'
-        env['READELF'] = join(toolchain, 'bin', android_host) + '-readelf'
-        env['STRIP'] = join(toolchain, 'bin', android_host) + '-strip'
-        env['STRIP'] += ' --strip-debug --strip-unneeded'
+        env['HOSTARCH'] = arch.command_prefix
+
+        env['CC'] = '{}'.format(
+            join(
+                self.ctx.ndk_dir,
+                'toolchains',
+                'llvm',
+                'prebuilt',
+                build_platform,
+                'bin',
+                arch.target + '-clang',
+            )
+        )
```

Comment:
Minor: why do you even `'{}'.format()` in here? You could directly return the `join()` result right?

Reply:
Yes, just fixed this, thanks @AndreMiras :heart: